bridge/configuration.php

```diff
@@ -0,0 +1,318 @@
+<?php
+
+// THE FOLLOWING FUNCTIONS ARE IMPLEMENTED AT THE C-level
+// \ddtrace_config_app_name()
+// \ddtrace_config_distributed_tracing_enabled()
+// \ddtrace_config_integration_enabled()
+// \ddtrace_config_trace_enabled()
+
+/**
+ * Reads and normalizes a string configuration param, applying default value if appropriate.
+ *
+ * @param string|null $value
+ * @param string $default
+ * @return string
+ */
+function _ddtrace_config_string($value, $default)
+{
+    if (false === $value || null === $value) {
+        return $default;
+    }
+
+    return trim($value);
+}
+
+/**
+ * Reads and normalizes a boolean configuration param, applying default value if appropriate.
+ *
+ * @param string|null $value
+ * @param boolean $default
+ * @return boolean
+ */
+function _ddtrace_config_bool($value, $default)
+{
+    if (false === $value || null === $value) {
+        return $default;
+    }
+
+    $value = strtolower($value);
+    if ($value === '1' || $value === 'true') {
+        return true;
+    } elseif ($value === '0' || $value === 'false') {
+        return false;
+    } else {
+        return $default;
+    }
+}
+
+/**
+ * Reads and normalizes a float configuration param, applying default value if appropriate.
+ *
+ * @param string|null $value
+ * @param float $default
+ * @param float $min if the final value is less then $min, then $min is returned.
+ * @param float $max if the final value is greater then $max, then $max is returned.
+ * @return float
+ */
+function _ddtrace_config_float($value, $default, $min = null, $max = null)
+{
+    if (false === $value || null === $value) {
+        return $default;
+    }
+
+    $value = strtolower($value);
+    if (is_numeric($value)) {
+        $floatValue = (float)$value;
+    } else {
+        $floatValue = (float)$default;
```

Comment:
This code is was literally lifted from `EnvVariableRegistry` or maybe a different Configuration class. The real point is that these are simply copy-pasted into simple functions. Luca did this part, but I suspect it's done this way so that it minimize risk of an unintentional change.